duffy-dispatch.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=runs worker as a service
+After=multi-user.target
+
+[Timer]
+OnCalendar=*-*-* *:*:00
+
+[Service]
+Type=idle
+User=duffy
+WorkingDirectory=/srv/duffy
+ExecStart=/srv/duffy2/venv/bin/python /srv/duffy/src/dispatch.py
```

Comment:
In production the venv is actually located at /srv/duffy2-venv/

We should fix that in the dev setup too so it matches